flowetl/deployment_example/README.md

```diff
@@ -0,0 +1,199 @@
+## FlowETL sample deployment
+
+This directory contains the files necessary to do an example deployment of FlowETL,
+which will ingest sample calls data from an external PostgreSQL database and sample
+SMS data from a CSV file.
+
+The remainder of this README describes the steps to set up this deployment and run
+the ingestion process. For the initial steps of building the relevant docker images,
+you need a local clone of the FlowKit repository. For simplicity, the steps below
+assume that you are running them from within this local clone, inside the
+directory `flowetl/deployment_example/`.
+
+In a production deployment, you would simply upload the docker images directly to the
+server, or pull them there if the server has internet connection. However, for development
+and testing it is useful to build the images locally, which is the method described here.
+Note that some other parts of the setup - e.g. using a docker swarm or setting up a
+local registry - are not strictly needed for a local example deployment, but we describe
+them here in order to mimic a production setup as closely as possible.
+
+
+### Set up a docker stack with FlowDB, FlowETL and an external PostgreSQL database
+
+First, set relevant environment variables. Most of them are pre-defined in `defaults.env`,
+but we need to set the path to the local clone of the FlowKit repo manually.
+
+```bash
+set -a && source ./defaults.env && set +a
+export LOCAL_FLOWKIT_REPO=$(pwd)/../..
+```
+
+Make the machine you're working on a docker swarm manager.
```

Comment:
```suggestion
Make the machine you're working on is a docker swarm manager.
```

Reply:
I think the original phrasing is already grammatically correct? Or are you suggesting to change it to "Make _sure_ the machine you're working on is a docker swarm manager."?